testnet/stacks-node/src/run_loop/neon.rs

```diff
@@ -515,7 +515,7 @@ impl RunLoop {
         let chain_state_db = self.boot_chainstate(burnchain_config);
 
         // NOTE: re-instantiate AtlasConfig so we don't have to keep the genesis attachments around
-        let moved_atlas_config = AtlasConfig::default(self.config.is_mainnet());
+        let moved_atlas_config = AtlasConfig::new(self.config.is_mainnet());
```

Comment:
This will need to use the configured atlas config:

```suggestion
        let moved_atlas_config = self.config.atlas.clone();
```